--- xapian-bindings-1.2.21/configure.ac~	2015-05-21 14:57:01.000000000 +0900
+++ xapian-bindings-1.2.21/configure.ac	2016-03-30 21:24:31.826127813 +0900
@@ -312,7 +312,7 @@
 	AC_MSG_RESULT(yes)
 	AC_MSG_CHECKING([for directory to install python bindings in])
 	if test -z "$PYTHON_LIB" ; then
-	  PYTHON_LIB=`$PYTHON -c 'import os,distutils.sysconfig;print(distutils.sysconfig.get_python_lib(1).replace(os.sep,"/"))'`
+	  PYTHON_LIB=`$PYTHON -c 'import os,distutils.sysconfig;print(distutils.sysconfig.get_python_lib(plat_specific=True,prefix="/usr").replace(os.sep,"/"))'`
 	fi
 	AC_MSG_RESULT([$PYTHON_LIB])
 	AC_ARG_VAR(PYTHON_LIB, [Directory to install python bindings in])
